src/integrations/windowsIntegrationManager.ts

```diff
@@ -420,13 +455,30 @@ export default class WindowsIntegrationManager implements IntegrationManager {
         },
         executable, 'kubeconfig', '--show');
 
+      console.log(`WSL distro "${ distro.name }: wsl-helper output: "${ stdout }"`);
```

Comment:
Should that be `debug` rather than `log`?

Reply:
I figured that it isn't called so frequently that making it `log` would make the logs congested, so it would be better to have access to this information without having to enable debug mode. But if you'd prefer `debug` I'm happy to change it.

Reply:
Wouldn't this be called every time the user goes to the integrations page (to fetch the list of integrations and their current states)?

Reply:
Yes, I believe it would be called several times (one for each distro) each time the user goes to the integrations page. Is that too much?